guide/14-deep-learning/how_retinanet_works.ipynb

```diff
@@ -0,0 +1,159 @@
+{
```

Comment:
<blockquote>It is also assumed that you have an understanding of object detection models. Please refer to the guide "<a href="https://developers.arcgis.com/python/guide/how-ssd-works/" target="_blank">How SSD Works</a>" where concepts such as anchor boxes and feature maps have been well explained.</blockquote><p> </p><p>In order to understand concepts like anchor box, grid cells, aspect ratio etc., please refer to the guide "[How SSD Works](https://developers.arcgis.com/python/guide/how-ssd-works/)”.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/562/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>The important concepts to learn from "How SSD Works" is the basics of object detection. That's why I mentioned "It is also assumed that you have an understanding of object detection models."</p><p> </p><p>I wouldn't mention grid cells because it doesn't apply directly to RetinaNet.</p>